File: Coursera Courses/Specializations/Programming in Java: A Hands-on Introduction/Java Object Basics: Functions, Recursion, and Objects/Week 3/LabChallenge.java
public class LabChallenge {
  public static void main(String[] args) {
    
    //add code below this line

    Dog dog1 = new Dog("Marceline", "German Shepherd");
    Dog dog2 = new Dog(dog1);
    dog2.name = "Cajun";
    dog2.breed = "Belgian Malinois";
    
    System.out.println(dog1.name + " " + dog1.breed);
    System.out.println(dog2.name + " " + dog2.breed);
    System.out.println(dog2 == dog1);

    //add code above this line
  }
}

File: Coursera Courses/Specializations/Programming in Python: A Hands-on Introduction/Python Object Basics: Functions, Recursion, and Objects/Week 3/lab_challenge.py
#Use the class below to solve the lab challenge
import copy

class Dog:
  def __init__(self, name, breed):
    self.name = name
    self.breed = breed
    
#Start your program below this line

dog1 = Dog('Marceline', 'German Shepherd')
dog2 = copy.deepcopy(dog1)
dog2.name = 'Cajun'
dog2.breed = 'Belgian Malinois'
